joplin/pages/base_page/models.py

```diff
@@ -52,14 +52,17 @@ class JanisBasePage(Page):
     publish_request_pk = models.TextField(blank=True, null=True)
```

Comment:
updating docstrings, except for at the bottom where the SEO panel is not added anymore